source/_integrations/devolo_home_network.markdown

```diff
@@ -47,6 +49,12 @@ Currently the following device types within Home Assistant are supported.
 - Restart the device
 - Start WPS
 
+### Images
+
+- QR code of your guest wifi credentials
+  - Updates every 10 seconds
```

Comment:
```suggestion
  - Updates every 15 seconds if changes are detected
```

Reply:
You're totally right ... The other sensors are wrong in the docs.